_maps/map_catalogue.txt

```diff
@@ -531,3 +531,45 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 		Size = (x = 35)(y = 21)(z = 1)
 		Tags = "No Combat", "Medium Loot", "Shelter"
 
+
+		Waste Ruins:
+
+		File name ="_maps\RandomRuins\wasteruins\wasteplanet_clowncrash.dmm"
+		Size = (x = 11)(y = 12)(z = 1)
+		Tags = "No Combat", "Minor Loot", "Shelter" "hospitable"
+
+		File name ="_maps\RandomRuins\wasteruins\wasteplanet_crashed_kitchen.dmm"
+		Size = (x = 11)(y = 16)(z = 1)
+		Tags = "Minor Combat Challenge", "Medium Loot", "hospitable"
+
+		File name ="_maps\RandomRuins\wasteruins\wasteplanet_fortress_of_solitude.dmm"
+		Size = (x = 45)(y = 42)(z = 1)
+		Tags =  "Minor Combat Challenge" "Medium Loot" "hospitable" "shelter"
+
+		File name ="_maps\RandomRuins\wasteruins\wasteplanet_lab.dmm"
+		Size = (x = 26)(y = 25)(z = 1)
+		Tags = "No Combat", "Medium Loot" "Shelter" "hazardous"
+
+		File name ="_maps\RandomRuins\wasteruins\wasteplanet_ore_processing_facility.dmm"
+		Size = (x = 30)(y = 26)(z = 1)
+		Tags = "No Combat", "Medium Loot" "Hazardous" "Lava"
+
+		File name ="_maps\RandomRuins\wasteruins\wasteplanet_pandora.dmm"
+		Size = (x = 18)(y = 21)(z = 1)
+		Tags = "Boss Combat Challenge", "Minor Loot" "Megafauna", "Shelter"
```

Comment:
Pandora doesn't protect from outside atmosphere. It can't be rated shelter. It's either inhospitable or hospitable depending on whether wasteplanet is liveable outside of storms

Reply:
swapped it for habitable